assets/templates/pages/base.html

```diff
@@ -38,5 +38,29 @@
     {% block inline_javascript %}
         {# Script tags with only code, no src (defer by default) #}
     {% endblock inline_javascript %}
+
+<script>
```

Comment:
Also, could you move this to the `head.html` template instead of `base.html`. Thank you.